files/usr/share/jeos-firstboot/modules/raspberrywifi

```diff
@@ -35,15 +35,23 @@ raspberrywifi_get_wlan_networks()
 {
 	list=()
 	local line
+	local has_hidden=false
```

Comment:
Looks like booleans, but isn't really. To avoid this trap:

```
local has_hidden=
...
has_hidden=1
...
if [[ ${has_hidden} ]]; then ...
fi
```